### PR TITLE
Submissions

### PR DESCRIPTION
**Pull request by ctlong**

### CODE REVIEW

File: app/assets/javascripts/angular/controller/static_pages/navbar.js.erb [hunk 1]
```diff
@@ -15,4 +15,18 @@ app.controller('NavbarCtrl', ['$scope', '$auth', '$location','$window', function
       console.log(response);
     });
   };
+
+  $scope.title = ''
+  switch($location.url().replace(/forms/,'').replace(/\d/,'')) {
+    case '///edit':
+      $scope.title = 'Design'
+      break;
+    case '///submissions':
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/static_pages/navbar.js.erb]: Why `///submissions`?

File: app/assets/javascripts/angular/controller/static_pages/navbar.js.erb [hunk 2]
```diff
@@ -15,4 +15,18 @@ app.controller('NavbarCtrl', ['$scope', '$auth', '$location','$window', function
       console.log(response);
     });
   };
+
+  $scope.title = ''
+  switch($location.url().replace(/forms/,'').replace(/\d/,'')) {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/static_pages/navbar.js.erb]: Are you trying to display the right title in Navbar, but they have different controllers, hence different $scope?

The correct way to communicate across controllers is this http://stackoverflow.com/questions/11252780/whats-the-correct-way-to-communicate-between-controllers-in-angularjs

**Comment by h8rry** [comment]
@ctlong please review

File: app/assets/javascripts/angular/controller/static_pages/navbar.js.erb (hunk 2 above)
Comment (h8rry) [inline app/assets/javascripts/angular/controller/static_pages/navbar.js.erb]: Alternative, you can use Angular services / factories for data persistency.

**Comment by ctlong** [comment]
@ggharry @f3r No more hacks (shoutout to fer).

**Pull request merged**